sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -399,7 +408,60 @@ export class NotificationActionsService {
     await this.notificationService.saveNotifications(
       [sinCompleteNotification],
       auditUserId,
-      entityManager,
+      { entityManager },
+    );
+  }
+
+  /**
+   * Create ECE Response file processing notifications.
+   * Currently multiple notifications are created as integration contacts
+   * can be multiple.
+   * TODO: Use email bulk send in GC Notify when bulk send email is available.
+   * @param notification notification.
+   */
+  async saveECEResponseFileProcessingNotification(
+    notification: ECEResponseFileProcessingNotification,
+  ): Promise<void> {
+    const auditUser = await this.systemUsersService.systemUser();
+    const templateId = await this.notificationMessageService.getTemplateId(
+      NotificationMessageType.ECEResponseFileProcessing,
+    );
+    const eceResponseFileProcessingNotifications = [];
+
+    for (const integrationContact of notification.integrationContacts) {
+      const messagePayload: EmailMessage = {
+        email_address: integrationContact,
+        template_id: templateId,
+        personalisation: {
+          date: this.getDateTimeOnPSTTimeZone(),
+          institutionCode: notification.institutionCode,
+          fileParsingErrors: notification.fileParsingErrors,
+          totalRecords: notification.totalRecords,
+          totalDisbursements: notification.totalDisbursements,
+          disbursementsSuccessfullyProcessed:
+            notification.disbursementsSuccessfullyProcessed,
+          disbursementsSkipped: notification.disbursementsSkipped,
+          duplicateDisbursements: notification.duplicateDisbursements,
+          disbursementsFailedToProcess:
+            notification.disbursementsFailedToProcess,
+          application_file: {
+            file: base64Encode(notification.attachmentFileContent),
```

Comment:
Just double checking, is there a scenario where the file would not be generated?
When everything goes well, are we including the summary also in the file?

Reply:
Yes, even when everything goes well we include the detailed summary in the file.